04-data-structures-part1.Rmd

```diff
@@ -339,6 +339,16 @@ In this case, we've explicitly told R that "control" should represented by 1, an
 "case" by 2. This designation can be very important for interpreting the
 results of statistical models!
 
+> ## Challenge 5 {.challenge}
+> Having to factors:
```

Comment:
This could be more explicit e.g. "The following code creates two variables of type factor:"